languages/go/exercises/concept/using-map/.docs/introduction.md

```diff
@@ -0,0 +1,10 @@
+`map` in go is a key-value type. In other programming language, you might know map as `dict` or associative array. The value of an uninitialized map is `nil`, and you can define map as follows
+
+`make(map[string]int)`
+
+or
+
+```go
+  var foo map[string]int
+  foo = make(map[string]int)
+```
```

Comment:
I love how concise you've kept the introduction! One thing I would add to the introduction too is the syntax to add/update value and to retrieve values. This is new syntax that is introduced in this exercise and thus should also be included here (see [this description](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)).